core-bundle/src/DependencyInjection/Configuration.php

```diff
@@ -690,4 +691,26 @@ static function (array $protocols): array {
             ->end()
         ;
     }
+
+    private function addRoutingNode(): NodeDefinition
+    {
+        return (new TreeBuilder('routing'))
+            ->getRootNode()
+            ->addDefaultsIfNotSet()
+            ->children()
+                ->arrayNode('override_dns')
+                    ->arrayPrototype()
+                        ->children()
+                            ->scalarNode('dns')
```

Comment:
DNS stands for "Domain Name System" and describes the overall "naming system used to identify computers reachable through IP networks". I would prefer either `domain` or `host` or `hostname`.

Reply:
On the other hand, `dns` is the name of the page field, that is being overwritten eventually.  `dns` sounds reasonable to me.